.github/workflows/CI.yaml

```diff
@@ -24,19 +42,16 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+
       - name: Build with Gradle
-        env:
-          TEAMCITY_DEV_URL: ${{ secrets.TEAMCITY_DEV_URL }}
-          TEAMCITY_USER: ${{ secrets.TEAMCITY_USER }}
-          TEAMCITY_PASSWORD: ${{ secrets.TEAMCITY_PASSWORD }}
-        run: ./gradlew build
-      - name: Clean neo4j from the cache
-        if: success() || failure() # run this step even if previous step failed
-        run: (find ~/.gradle/caches -name "*neo4j*" -exec rm -rf {} \;) || echo "All neo4j files cleaned"
-      - name: Check procedures included in apoc full
-        run: git diff --exit-code full/src/main/resources/extended.txt || (echo "extended.txt file does not contain all changes" && exit -1)
+        run: chmod +x gradlew && ./gradlew build
```

Comment:
Simplify these lines and make sure there's no `build --continue` in other branches.